### ISSUE TITLE
Failing test suite

### ISSUE THREAD

**Issue by simonihmig** [MartinMalinda/ember-hiding-menu#6]
I see that most tests are failing here, because on the one hand you still use jQuery in your code, but on the other hand you are using ember-native-dom-event-dispatcher and have removed jquery from the build. So what's the idea? :)

I wanted to fix some things, because I was seeing the following errors in our tests, wich are also happening in this addon (after enabling jQuery again):

`Global error: Error: Assertion Failed: You have turned on testing mode, which disabled the run-loop's autorun. You will need to wrap any code with asynchronous side-effects in a run`

**Comment by MartinMalinda** [comment]
@simonihmig I agree, the addon is not in a happy place right now. I kind of rushed last releases and the tests are behind, I tested it in my parent app only. The parent app uses `Zepto` instead of `JQuery` that's why there's JQuery like syntax in the addon without the actual JQuery.

My parent app still needs a few addons that use JQuery syntax, that's why I opted for Zepto and I've monkey patched a few missing APIs of it to make it work.

I will refactor to vanilla JS and fix tests soon.. hopefully :)
